docs/workshop/part-ii/01-writing-your-first-promise.mdx

```diff
@@ -270,43 +265,71 @@ In a nutshell, a Kratix `Pipeline` is a series of containers that will be execut
 
 When a user requests a new App deployment, the `resource.configure` Workflow will be executed. This workflow will query the user's input and declare the resources that must be created.
 
-Start by creating a new directory called `workflows` inside the `app-promise` directory. Then, inside the directory, create a `Dockerfile`. The Dockerfile will define the container image that will be executed as part of the `resource.configure` Workflow:
+We will once again use the Kratix CLI to create the necessary files. Run the following command to bootstrap the skeleton of your Workflow:
 
-```bash
-mkdir -p workflows
-touch workflows/Dockerfile
+```
+kratix add container resource/configure/mypipeline --image kratix-workshop/app-pipeline-image:v1.0.0
 ```
 
-Open the `Dockerfile` in your editor and add the following content:
+The command above should have created a directory structure that looks like the following:
 
-```mdx-code-block
-import Dockerfile from "!!raw-loader!./_partials/writing-a-promise/03-Dockerfile"
+```
+.
+├── README.md
+├── dependencies
+│   └── dependencies.yaml
+├── example-resource.yaml
+├── promise.yaml
+└── workflows
+    └── resource
+        └── configure
+            └── mypipeline
+                └── kratix-workshop-app-pipeline-image
+                    ├── Dockerfile
+                    ├── resources
+                    └── scripts
+                        └── pipeline.sh
 ```
 
-<CodeBlock language="dockerfile" title="app-promise/workflows/Dockerfile">
-    {Dockerfile}
-</CodeBlock>
+The CLI will create a basic `Dockerfile` and a `pipeline.sh` script. We will customise these files to fit our needs as we progress through the workshop. Feel free to explore them.
 
-The Dockerfile above uses the `kratix-pipeline-utility` image as a base image. This image provides a set of utilities that make it easier to write Kratix Pipelines. You may also notice that image will execute a script called `resource-configure` when the container starts.
+The generated Dockerfile uses the `kratix-pipeline-utility` image as a base image. This image provides a set of utilities that make it easier to write Kratix Pipelines. You may also notice that image will execute a script called `pipeline.sh` when the container starts.
```

Comment:
Not sure what you mean by base image, generated dockerfile I see:
```
❯ cat workflows/resource/configure/mypipeline/kratix-workshop-app-pipeline-image/Dockerfile
FROM "alpine"

RUN apk update && apk add --no-cache yq

ADD scripts/pipeline.sh /usr/bin/pipeline.sh
ADD resources resources

RUN chmod +x /usr/bin/pipeline.sh

CMD [ "sh", "-c", "pipeline.sh" ]
ENTRYPOINT []
```

Reply:
yep, that was a miss! resolved!